packages/near-snap/src/lib/icon.ts

```diff
@@ -1 +1 @@
p/WGM+vuyKawcIBuBP2z4xd07JiV5grL3QGHuhlZk3yj/Z3Ll6yzFUroIEAMKVALix80sy+p1D/jgvoyeNtY9YYx7pXLXlJ3SOQHBYa82Ge++8wUp/KylNRIAJGzDG/t6SK69nh3AgSM9ByWz5ZOevGGsvtMZcKKuzJSUO+aJ/6vr8lo8RLZAAQKhs/eT8hZ41X5fULqPHrWcecdPehvndW7NEBwi2dd+4863G2K9JOp1oAOMdnZlHKmXnuvOv/sgWggEEfLzbPT9VyTjnGsdeKKvzJA06xn5o/qqtm4gOSAAAAFonCbCuO+LsPuZPrLV/pfoepQS0ipyx5tPnrrjmVmMM1W8AABIAAIBg2fDNf32P55mvSeZEogGMOCD7gWPcD59z5UdeIhoAgHpxCAEAoJ7OveK3fhirtJ8qq1vFnh7AocpWWpXeUTqHyT8AoN6oAAAANMz6u2+/0DrmTknziQagX3iO85vnXXHNs4QCANAIVAAAABpmyVXXP2Idc7KM/pNoIMSspC+3xUqnM/kHADQSFQAAgKZ44t47r5K1/yJp+kHTIp5MaG2bjNG1S6687nFCAQBoNCoAAABNsfTKa++uyHunjJ7Y/4dM/tHa7o5VvHcy+QcANAtDLQBAU1lrzYZ7vvr71thVkuJEBC1owBj7e0uuvP7rhAIAQAIAABB6j3/jqyc7xvuapHcSDbTQQOvhcsW9/vyrP7KFaAAASAAAADDsmX/5l+jgrMifGpnPSnKJCAIsZ6z59LkrrrnVGMPxlwAAEgAAABzJ+rvvONM6+ndJi4kGAji4+oFj3A+fc+VHXiIaAAA/YRNAAIDvLLnquqcTudhpkr5MNBAgZSv7F9srbWcx+QcA+BEVAAAAX3v8G7evcIz5Z0mziAZ8PKL6uWecD593xTXPEgwAgF9RAQAA8LXzPnj9PeVI5e2S7iMa8CEr6ctt0dK7mfwDAPyOCgAAQGCsv/f2D1tr/lFSG9GAD2yS7DVLV1y/jlAAAEgAAABQ6yTAXV851rruv0s6m2igie524/GPnnPZh3YTCgAACQAAAOpk3bruiLP7mD+x1v6lpBgRQQMNGJnfXbLi2v9HKAAAJAAAAGiQJ+65890y9muyegvRQAMGTQ+7EXvd2e+/fivRAACQAAAAoMGevuuuZMEd+r9GulFsbov6yBlrPn3uimtuNcZYwgEAIAEAAEATPXHvnRfI2jsldRIN1ND3jTUfXvLBa18mFACAoONNCQCgJSy98tpHrWPebmW+TjRQA2Ur+xd9lbazmfwDAFoFFQAAgJbzxL13XiVr/1nSDKKBCYyOfu5Y/ea5K657jmAAAEgAAADgc0/e9e8Ly275q5KWEQ2MkZX0lbZY6Y9Ov/yjWcIBACABAABAUGZz1poN9955g5VukZQiIjiKTZK9ZumK69cRCgAACQAAAAJqw113vM1z9TVJpxENHMHdbjz+0XMu+9BuQgEAIAEAAEDAvXDXXbE+d+gvjfQJSS4RgaR+yX506Yrrv0koAAAkAAAAaDHr7r39bGOdf5fssUQj1O4rRyo3LH//DdsJBQCABAAAAC3qyW/f3l4umy9I+m2iETpZY81nzl1xza3GGEs4AAAkAAAACIHH7/7q+Y5TOc/KsCQgDKwdkOv817Irrt1IMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABg7P5/hOtVkRNB7WQAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjItMDgtMzFUMTI6MDM6NDMrMDA6MDB1WYO7AAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIyLTA4LTMxVDEyOjAzOjQzKzAwOjAwBAQ7BwAAAABJRU5ErkJggg==`;
```

Comment:
Can you please reduce the size of this new icon as it increases the bundle size of the package?

Before this update:
```bash
  index.js 67.713 KB
  index.cjs 67.795 KB

```

After this update:
```bash
  index.js 120.134 KB
  index.cjs 120.223 KB
```

Reply:
Also the icon in the `assets` folder needs to be updated too: https://github.com/AZbang/wallet-selector/blob/update-near-snap/packages/near-snap/assets/near-icon.png